### ISSUE TITLE
Application after upgrading to v 6.1.0 doesn't work

### ISSUE THREAD

**Issue by tolsam** [oblique-bit/oblique#22]
After upgrading Oblique from 6.0.1 to 6.1.0, our application doesn't compile with the following error
```
Error: ./node_modules/@oblique/oblique/__ivy_ngcc__/fesm2015/oblique.js
Module not found: Error: Can't resolve '../assets/obliqueIcons.svg' in
'C:\project\src\main\angular\infofito\node_modules\@oblique\oblique\__ivy_ngcc__\fesm2015'
```
 Thank you in advance

**Comment by c-saner** [comment]
Hi tolsam,

We are sorry for the inconvenience.

Unfortunately, we noticed this error on 06.04.2021 after the release was announced.

We are still analyzing the problem.

The easiest (temporary) solution would be to downgrade to Oblique 6.0.1. And update to Oblique 6.1.X when we have delivered a fix.

We will inform you as soon as we have delivered a fix.

Thank you very much for your understanding.

Kind regards
